pom.xml

```diff
@@ -17,7 +17,7 @@
 			<dependency>
 				<groupId>edu.wisc.my.apps</groupId>
 				<artifactId>uw-frame</artifactId>
-				<version>2.3.1</version>
+				<version>2.5.0</version>
```

Comment:
As of 10 minutes ago, 2.5.2 was the latest. Should prob just upgrade right to that.